content/packit-as-a-service.md

```diff
@@ -4,31 +4,33 @@ date: 2019-06-22
 draft: false
 ---
 
-Packit-as-a-Service is a GitHub App (https://developer.github.com/apps/about-apps/).
+Packit-as-a-Service is a [GitHub App](https://developer.github.com/apps/about-apps/)
+that helps you continuously ensure that your project work in Fedora OS.
 
 ### Easy to use
 
-Using Packit Service is very straightforward: add one config file to your repository
-([see .packit.yaml format](https://github.com/packit-service/packit/blob/master/docs/configuration.md)),
-followed by an RPM spec file and you're good to go. Besides the service, packit is also a tool
-([see packit README.md](https://github.com/packit-service/packit/blob/master/README.md)),
-so you can always try things locally on your own.
+Using Packit Service is very straightforward: add one [config
+file](https://github.com/packit-service/packit/blob/master/docs/configuration.md)
+to your repository together with the RPM spec file and you're good to go. 
 
 Packit Service validates your pull requests by building your software in Fedora OS.
-Once the builds are done, packit lets you know how to install the
+Once the builds are done, Packit lets you know how to install the
 RPMs inside your environment so you can try the changes before merging them.
 
+Packit is also available as a [CLI tool](https://github.com/packit-service/packit/blob/master/README.md),
```

Comment:
I would prefer to use a tool. We have had a discussion with @jpopelka about it (https://github.com/packit-service/packit.dev/pull/15#discussion_r297253118)

Reply:
@jpopelka why not mention the CLI tool here?

Reply:
It read "tool" before which is pretty much the same as CLI tool. But let me know and I remove it completely.

Reply:
I'm personally +1 on using the wording "CLI tool".